src/surveys/components/ChoiceResponse.tsx

```diff
@@ -81,31 +74,31 @@ export default function SelectResponse({
       />
 
       <div className={'space-y-2 min-w-[381px]'}>
-        {questionOptions.map((option) => (
-          <label
-            key={option.id}
-            className={'rounded-xl py-3 px-4 flex gap-2 items-center'}
-            style={{ backgroundColor: theme?.answer ?? '#F0F0F0' }}
-          >
-            <input
-              type={
-                question.type === AnswerType.MULTIPLE ? 'checkbox' : 'radio'
-              }
-              value={option.id}
-              checked={
-                question.type === AnswerType.MULTIPLE
-                  ? (selectedOption as ID[]).includes(option.id)
-                  : selectedOption === option.id
-              }
-              onChange={() =>
-                question.type === AnswerType.MULTIPLE
-                  ? handleOptionChange(option.id)
-                  : setSelectedOption(option.id)
-              }
-            />
-            <span>{option.label}</span>
-          </label>
-        ))}
+        {questionOptions &&
+          questionOptions.map((option) => (
+            <label
+              key={option.id}
+              className={'rounded-xl py-3 px-4 flex gap-2 items-center'}
+              style={{
+                backgroundColor: theme?.answer
+                  ? hexToRgba(theme.answer, 0.1)
+                  : '#F0F0F0',
+                color: theme?.answer ?? '#050505',
+              }}
+            >
+              <input
+                type={
+                  question.type === AnswerType.MULTIPLE ? 'checkbox' : 'radio'
```

Comment:
@afolasope It doesn't look like you've updated the styling of the radio buttons/checkboxes

Reply:
No, I'm still working on it